meta/data/login.seed.json

```diff
@@ -0,0 +1,25 @@
+[{
+	"_id": {
+		"$oid": "597db4308583ff17ac43ef52"
+	},
+	"email": "joe.doe@foo.com",
```

Comment:
I think it would be a good idea to create fake users easy to rememeber) per rol as well, something like

```
admin@sample.com
trainer@sample.com
student@sample.com
```

We can update the mocked front api as well